### PR TITLE
docs(faq): info about wallet import via UI

### PR DESCRIPTION
**Pull request by theborakompanioni**
Adds the following section to the FAQ page:
> Yes, importing an existing wallet can be done via the web interface since Jam `v0.1.6`.
Just make sure you are running JoinMarket `v0.9.10` or later.

The previously existing paragraph has been moved to "How to import wallets via the command line?".

### CODE REVIEW

**Comment by theborakompanioni** [comment]
ping @dergigi

File: docs/FAQ.md
Comment (MarnixCroes) [inline docs/FAQ.md:169]: maybe add a line how to do this?
Where can the user find this?

File: docs/FAQ.md
Comment (MarnixCroes) [inline docs/FAQ.md]: nit
```suggestion
### How to import an existing wallet via the command line?
```

File: docs/FAQ.md
Comment (theborakompanioni) [reply docs/FAQ.md:169]: Done. Do you think it should contain additional information?

Comment (MarnixCroes) [reply docs/FAQ.md:169]: Maybe as a follow-up, import wallet could have a section with detailed information imo, yes.
With all ins and outs.

Comment (theborakompanioni) [reply docs/FAQ.md:169]: I think this is a good idea. However, the import process is quite straight forward, so I'd wait for someone to come and want to have it explained in more detail. Okay with you?

Thanks, brother :pray:

Comment (MarnixCroes) [reply docs/FAQ.md:169]: Sure

**Review by MarnixCroes — APPROVED** [review]

**Comment by MarnixCroes** [comment]
what's up with this?
can't be merged? :D

**Comment by theborakompanioni** [comment]
> what's up with this? can't be merged? :D

Hey @MarnixCroes, thanks for asking and the reminder.
It needs approval from one of the maintainers, e.g. @dergigi.
My hands are tied.

**Comment by MarnixCroes** [comment]
I see.
maybe as a work around, I can cherry-pick your commit and create a PR. 
If there is still no reply from gigi (in a week?).
as you can merge other's pr's but not your own right.

**Comment by theborakompanioni** [comment]
> [...] I can cherry-pick your commit and create a PR. [...] as you can merge other's pr's but not your own right.

I am not sure. It would be strange if that worked. But I am thrilled by your creativity. : D

Edit: Yes, it will work. Just verified with your PR #51. :raised_hands:

**Comment by theborakompanioni** [comment]
Superseded by #52.
Thanks you @MarnixCroes :pray:

**Pull request closed without merging**
